LiskHQ/lisk-core
# Fix registerKeys command implementation

### What was the problem?

This PR resolves #757 

### How was it solved?

- [x] Update the registerKeys command implementation
- [x] Regenerate genesis block for alphanet/betanet/devnet
- [x] Update unit tests (Additional unit tests refactoring will be done as a part of issue #759 )

### How was it tested?
- ./bin/run start -n alphanet
- Create legacy registerKeys transaction
- Enable generators on alphanet
  - ./bin/run keys import -f ./config/alphanet/dev-validators.json
  - ./bin/run endpoint invoke generator_setStatus '{ "address": "validatorAddress", "height": 0, "maxHeightGenerated": 0, "maxHeightPrevoted": 0 }'
  - ./bin/run generator enable validatorAddress --height=0 --max-height-generated=0 --max-height-prevoted=0  
- Check for `isBanned` flag in validators response  (it should be false)
